apps/meteor/app/lib/server/functions/sendMessage.ts

```diff
@@ -212,14 +213,71 @@ export function prepareMessageObject(
 	}
 }
 
+const parseFilesIntoMessageAttachments = async (
+	filesToConfirm: Partial<IUpload>[],
+	roomId: string,
+	user: IUser,
+): Promise<{ files: FileProp[]; attachments: MessageAttachment[] }> => {
+	const messageFiles: FileProp[] = [];
+	const messageAttachments: MessageAttachment[] = [];
+
+	const fileAttachmentPromises = filesToConfirm.map(async (file) => {
+		try {
+			if (!file) {
```

Comment:
You could use the filter method first in the chain

Reply:
yes sir i have added the filter to check the file